### PR TITLE
[4.3] add CustomNoUpgrade to FeatureSets and LatencySensitive featuresets

### PR DESCRIPTION
**Pull request by rphillips**
Adds CustomNoUpgrade and LatencySensitive to the Features map.

/cc @sjenning @deads2k

### CODE REVIEW

**Comment by rphillips** [comment]
force pushed a tweak to not duplicate the default feature gates.

**Comment by rphillips** [comment]
@deads2k could you review this again since 4.3 has opened?

File: config/v1/types_feature.go [hunk 1]
```diff
@@ -95,7 +109,9 @@ type FeatureGateEnabledDisabled struct {
 //
 // If you put an item in either of these lists, put your area and name on it so we can find owners.
 var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
-	Default: {
+	Default:         defaultFeatureGates,
+	CustomNoUpgrade: defaultFeatureGates,
```
Comment (deads2k) [inline config/v1/types_feature.go]: I find this suspicious.  I think custom should be custom.  Maybe they want to adjust something we set by default. 

Please remove the change from this pull

File: config/v1/types_feature.go [hunk 2]
```diff
@@ -35,6 +35,9 @@ var (
 	// Because of its nature, this setting cannot be validated.  If you have any typos or accidentally apply invalid combinations
 	// your cluster may fail in an unrecoverable way.
 	CustomNoUpgrade FeatureSet = "CustomNoUpgrade"
+
+	// TopologyManager enables ToplogyManager support. Upgrades are enabled with this feature.
+	TopologyManager FeatureSet = "TopologyManager"
```
Comment (deads2k) [inline config/v1/types_feature.go]: @derekwaynecarr this name looks odd.  Is this related to high maintenance, high performance clusters?

Comment (rphillips) [reply config/v1/types_feature.go]: I talked to @sjenning about this and he mentioned that if it's called 'High Performance', then everyone might see that and want to turn it on.

File: config/v1/types_feature.go (hunk 1 above)
Comment (rphillips) [reply config/v1/types_feature.go]: Fixed the CustomNoUpgrade up. Thanks

File: config/v1/types_feature.go (hunk 2 above)
Comment (rphillips) [reply config/v1/types_feature.go]: There is some talk about High Performance using a RT kernel as well, so whatever name we come up with should describe the feature.

Comment (derekwaynecarr) [reply config/v1/types_feature.go]: alternative names:

- performance sensitive
- latency sensitive

@deads2k do you see an issue if in future we let you transition from "LatencySensitive" to "Default" set?

Comment (deads2k) [reply config/v1/types_feature.go]: > @deads2k do you see an issue if in future we let you transition from "LatencySensitive" to "Default" set?

I think I'm more inclined to change the meaning of default to match "LatencySensitive". 

LatencySensitive works. PerformanceSensitive will be one that everyone tries to turn on, so I would avoid it.

**Comment by rphillips** [comment]
Ready for review. Updated to support LatencySensitive. Thank you!

**Pull request merged**